CHANGELOG.md

```diff
@@ -48,7 +48,7 @@
 - PR #3138 Movey unary files to legacy
 - PR #3175 Set cmake cuda version variables
 - PR #3171 Move deprecated error macros to legacy
-
```

Comment:
This should just be an addition of a line, not also a removal of a whitespace line.

Reply:
Done.  PTAL.